### ISSUE TITLE
ncm-metaconfig: Add ceph client configuration

### ISSUE THREAD

**Issue by alvarosimon** [quattor/configuration-modules-core#1277]
We need to set the ceph client configuration into ` ceph.conf`  . This is done already by Ceph component but not for the clients.

OpenStack Manila server requires this kind of setup from `ceph.conf`   (besides `mon_host` endpoints):

```
[client.manila]
client mount uid = 0
client mount gid = 0
log file = /var/log/manila/ceph-client.manila.log
admin socket = /var/run/manila/ceph-$name.$pid.asok
keyring = /etc/ceph/ceph.client.manila.keyring
```

**Comment by Pansanel** [comment]
I agree with your request. We should move the ```config.pan``` to ```sysconfig.pan``` and create a config.pan with an extensive schema for ceph.conf. I will to work on it, _now_.
